tests/Integration/Fixtures/Synonyms.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Meilisearch\Bundle\Tests\Integration\Fixtures;
+
+class Synonyms
```

Comment:
Besides agreeing with the current implementation. I think it would be more idiomatic to the developers if we define a common interface through those services. What do you think?

This interface will enforce the methods, and we could use `instanceof` in the implementation instead of `is_callable($value['_service'])`.

Reply:
Do you have a name for the interface? :wink:

Reply:
Implemented as `interface SettingsProvider`